.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
```

Comment:
I think we should add the `open-pull-requests-limit: 10` configuration to limit the number of open PRs for easy access and maintenance.

Reply:
the default is already limited to 5 PRs, so I think we are good (see https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#open-pull-requests-limit)

Reply:
Good to know, thanks :+1: